File: server/lib/fakers_api/people/person_address.ex
defmodule FakersApi.People.PersonAddress do
  use Ecto.Schema
  import Ecto.Changeset
  alias FakersApi.People.{Person, Address}

  @primary_key false
  schema "person_address" do
    field :assigned, :date
    field :is_primary, :boolean

    belongs_to :person, Person, primary_key: true
    belongs_to :address, Address, primary_key: true
  end

  @doc false
  def changeset(person_address, attrs) do
    person_address
    |> cast(attrs, [:assigned, :person_id, :address_id, :is_primary])
    |> unique_constraint(:composite_key, name: :person_address_pkey)
  end
end

File: generator/EntityGenerator/model/PersonAddress.cs
using System;

#nullable disable

namespace EntityGenerator.model
{
    public partial class PersonAddress
    {
        public long PersonId { get; set; }
        public long AddressId { get; set; }
        public DateTime Assigned { get; set; }

        public virtual Address Address { get; set; }
        public virtual Person Person { get; set; }
        
        public override string ToString() => $"{PersonId} {AddressId} {Assigned}";
    }
}